ush/minmon_xtrct_reduct.pl

```diff
@@ -77,7 +77,7 @@
 
    if( -e $outfile ) {
       my $newfile = "${tankdir}/${outfile}";
-      system("cp -f $outfile $newfile");
+      system("cpfs -f $outfile $newfile");
```

Comment:
```suggestion
      system("cpfs $outfile $newfile");
```
Does `cpfs` take `-f`?

Reply:
No it does not.

Reply:
Fixed.